socorro/external/es/supersearch.py

```diff
@@ -195,6 +192,12 @@ def get(self, **kwargs):
         The list of accepted parameters (with types and default values) is in
         the database and can be accessed with the super_search_fields service.
         """
+        # Require that the list of fields be passed.
+        if not kwargs.get('_fields'):
+            raise MissingArgumentError('_fields')
+        self.all_fields = kwargs['_fields']
+        self.build_fields()
```

Comment:
You might want to consider calling this `self._build_fields()` because it's an implementation detail of this class that is boring to outside implementers of the class.